interceptor/tracing_interceptor.go

```diff
@@ -788,20 +818,71 @@ func (t *tracingWorkflowOutboundInterceptor) startNonReplaySpan(
 		},
 		ToHeader: true,
 		Time:     time.Now(),
-	})
+	}, t.root.workflowHeaderReader(ctx), headerWriter)
 	if err != nil {
-		return nopSpan{}, ctx, newErrFut(ctx, err)
+		return nopSpan{}, ctx, workflowFutureFromErr(ctx, err)
 	}
 	return span, newCtx, nil
 }
 
+type tracingNexusOperationInboundInterceptor struct {
+	NexusOperationInboundInterceptorBase
+	root *tracingInterceptor
+}
+
+// CancelOperation implements internal.NexusOperationInboundInterceptor.
+func (t *tracingNexusOperationInboundInterceptor) CancelOperation(ctx context.Context, input NexusCancelOperationInput) error {
+	info := nexus.ExtractHandlerInfo(ctx)
+	// Start span reading from header
+	span, ctx, err := t.root.startSpanFromContext(ctx, &TracerStartSpanOptions{
+		Operation:  "RunCancelNexusOperationHandler",
```

Comment:
There are all sorts of other name options we can bikeshed over here (`RunCancelNexusOperation`, `HandleNexusCancelOperation`, etc), but so long as consistent across SDKs, I don't have a strong preference.